next.config.js

```diff
@@ -6,6 +6,9 @@ const nextraDocsConfig = {
 };
 
 const nextJsConfig = {
+  env: {
+    SEGMENT_KEY: process.env.SEGMENT_KEY
```

Comment:
I'm surprised you do this. It should just come from the environment without needing to "redeclare it". Certainly can't harm tho.

Reply:
This is how you get it to get passed to the client.  Environment is only usable on the serverside by default